README.md

```diff
@@ -90,6 +90,41 @@ Please note that if you use a kubernetes cluster created on DigitalOcean, there
 will be a cloud controller manager running in the cluster already, so you local
 one will compete for API access with it.
 
+### Run Locally (optional parameters)
+
+In addition to the standard way of running `digitalocean-cloud-controller-manager`
+locally against a particular cluster, you can add some additional parameters that
+enable you to run the cluster with some extra features.
+
+If you want to include a `PUBLIC_ACCESS_FIREWALL_NAME` and `PUBLIC_ACCESS_FIREWALL_TAGS` for your cluster. You can run the following command.
+
+```bash
+cd cloud-controller-manager/cmd/digitalocean-cloud-controller-manager
+FAKE_REGION=fra1 DO_ACCESS_TOKEN=your_access_token PUBLIC_ACCESS_FIREWALL_NAME=firewall_name       \
+PUBLIC_ACCESS_FIREWALL_TAGS=k8s,k8s:8847f3c3-387f-4795-b794-b2f7d9f7d22f,k8s:worker go run main.go \
+  --kubeconfig <path to your kubeconfig file>                                                      \
+  --leader-elect=false --v=5 --cloud-provider=digitalocean
+```
+
+If you want to expose prometheus metrics, be sure to include the endpoint by
+passing it in as the `METRICS_ADDR`.
+
+```bash
+cd cloud-controller-manager/cmd/digitalocean-cloud-controller-manager
+FAKE_REGION=fra1 DO_ACCESS_TOKEN=your_access_token METRICS_ADDR=<host:port> go run main.go      \
+  --kubeconfig <path to your kubeconfig file>                                                   \
+  --leader-elect=false --v=5 --cloud-provider=digitalocean
+```
+
+After you have started up the CCM, run the following curl command if you want to view the prometheus metrics output. In order to illustrate the example, we'll assume you've set `METRICS_ADDR` to `localhost:8080`.
```

Comment:
s/if you want//

Also you should consider sticking with the same placeholder value (`<host:port>`) for `METRICS_ADDR` as what you used in the `go run main.go` example above; then you don't need to say "In order to illustrate the example, we'll assume you've set `METRICS_ADDR` to `localhost:8080`." at all